client/src/API/index.js

```diff
@@ -134,6 +134,36 @@ const fetchUserPosts = async ({
   }
 };
 
+const upvoteAPost = async (postId, userId, type) => {
+  let response;
+  try {
+    response = await axios.patch(`${axiosConfig.baseURL}/posts/${postId}/${type}`, { userId }, axiosConfig);
+  } catch (err) {
+    return alert(err.response.data.message);
```

Comment:
Don't use alert here. Throw err in the catch block and catch that err in where you are calling thi function. Use alert there. So we can change alerts to modal.
Check signin/sign-up function for reference

Reply:
done